src/System.ServiceModel.Primitives/tests/ServiceModel/DuplexClientBaseTest.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Threading.Tasks;
+using Xunit;
+
+public class DuplexClientBaseTest
+{
+    [Fact]
+    public static void DuplexClientBase_Ctor_Initializes_State()
+    {
+        InstanceContext context = new InstanceContext(new WcfDuplexServiceCallback());
+        Binding binding = new NetTcpBinding();
+        EndpointAddress endpoint = new EndpointAddress(Endpoints.Tcp_NoSecurity_Callback_Address);
+        MyDuplexClientBase<IWcfDuplexService> duplexClientBase = new MyDuplexClientBase<IWcfDuplexService>(context, binding, endpoint);
+
+        Assert.Equal<EndpointAddress>(endpoint, duplexClientBase.Endpoint.Address);
+        Assert.Equal<CommunicationState>(CommunicationState.Created, duplexClientBase.State);
+
+        duplexClientBase.Abort();
+    }
```

Comment:
Also, we as a rule don't use `using` when dealing with WCF service proxies - it's well documented (see [http://blogs.msdn.com/b/jjameson/archive/2010/03/18/avoiding-problems-with-the-using-statement-and-wcf-service-proxies.aspx](example)). 

While in CoreCLR we can probably (finally) change this behaviour, we have to be mindful that in-contract code working in CoreCLR must work in Desktop... so if we start using `using` statements here... well, see above about "sample code" :smile: